ZaeXT/test-scripts/test_backend.py

```
@@ -0,0 +1,459 @@
+    print("🚀  开始对AI对话系统后端进行全功能自动化测试...")
+    try:
+        test_user_flow()
+        test_models_and_categories_flow()
+        test_conversation_and_chat_flow()
+        # --- 新增的调用 ---
+        test_memory_and_multiturn_flow()
+        
+        test_recycle_bin_flow() # 确保回收站测试在最后，因为它会删除一个重要对话
+        
+        print("\n" + "="*50)
+        print("  🎉  \033[92m所有测试均已成功通过！\03d[0m")
```

Ruff (0.14.1)

String contains ambiguous `！` (FULLWIDTH EXCLAMATION MARK). Did you mean `!` (EXCLAMATION MARK)?

(RUF001)